packages/react-html/README.md

```diff
@@ -12,112 +12,267 @@ $ yarn add @shopify/react-html
 
 ## Usage
 
-The `<HTML>` component serves as a top level wrapper for a react application, allowing you to avoid needing any kind of server side template, in favor of purely using `reactDom.renderToString`.
+This PR exposes two entrypoints:
 
-```javascript
-import * as React from 'react';
-import {renderToString} from 'react-dom/server';
+- `@shopify/react-html`: which contains code that can be used on server and client. Code in your `app` and `client` directories should import from this entrypoint.
```

Comment:
keep in mind that this is for general consumption. Not all users will have an `app` or `client` directory.

Reply:
Perhaps this:

> `@shopify/react-html`: which contains code that can be used on server and client. Code executed in a browser context (for example, an `app` or `client` directory) should import from this entrypoint.

Reply:
I think it is fine to leave as-is, only Node apps can currently use this. Will leave a note to that effect.